lib/tizen_doctor.dart

```diff
@@ -23,15 +27,11 @@ class TizenDoctorValidatorsProvider extends DoctorValidatorsProvider {
   List<DoctorValidator> get validators {
     final List<DoctorValidator> validators =
         DoctorValidatorsProvider.defaultInstance.validators;
-    for (final DoctorValidator validator in validators) {
-      // Append before any IDE validators.
-      if (validator is AndroidStudioValidator ||
-          validator is NoAndroidStudioValidator) {
-        validators.insert(validators.indexOf(validator), tizenValidator);
-        break;
-      }
-    }
-    return validators;
+    return <DoctorValidator>[
```

Comment:
How about this? I think it is more succinct.
```dart
    final List<DoctorValidator> validators = List<DoctorValidator>.from(
        DoctorValidatorsProvider.defaultInstance.validators);
    validators.insert(1, tizenValidator);
    return validators;
```

Reply:
Hmm. I don't get the point.

Reply:
I think it's easier to read, you can ignore this comment~